eZ/Bundle/EzPublishCoreBundle/Resources/config/papi.yml

```diff
@@ -75,3 +75,31 @@ services:
             - '@ezpublish.spi.search'
         tags:
             - { name: kernel.event_subscriber }
+
+    eZ\Publish\Core\FieldType\Image\ImageThumbnailStrategy: ~
+    eZ\Publish\Core\FieldType\ImageAsset\ImageAssetThumbnailStrategy:
+        lazy: true
+        arguments:
+            $thumbnailStrategy: '@eZ\Publish\Core\Repository\Strategy\ContentThumbnail\ThumbnailChainStrategy'
+            $contentService: '@ezpublish.api.service.content'
+
+    eZ\Publish\Core\Repository\Strategy\ContentThumbnail\Field\ContentFieldStrategy:
+        arguments:
+            $strategies:
+                ezimage: '@eZ\Publish\Core\FieldType\Image\ImageThumbnailStrategy'
+                ezimageasset: '@eZ\Publish\Core\FieldType\ImageAsset\ImageAssetThumbnailStrategy'
+
+    eZ\Publish\Core\Repository\Strategy\ContentThumbnail\StaticStrategy:
+        arguments:
+            $staticThumbnail: 'https://via.placeholder.com/150'
```

Comment:
This is rather risky as we have no control over it. It should be rather configurable and set in the meta-repository